### DISCUSSION TITLE
Notifications and UI for viewing all overdue loans

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5612] [Feature Requests]
There should be some way to easily view all overdue loans from the Interactions menu. 

These could be grouped by agent. It would be nice to have the ability to prioritize certain loans, sort them, and introduce a way to notify these institutions (#1952).

Managing these loans is currently really difficult and it is easy to have them fall into the background. 

Requested By: Ana (Herpatology Collection Manager at KU)

**Comment by maxpatiiuk** [comment]
@grantfitzsimmons please ask Ana what issue she had with using query builder for this.

**Comment by grantfitzsimmons** [comment]
I have yet to recieve a reply @maxpatiiuk

**Comment by maxpatiiuk** [comment]
Partially possible though the stats page

**Comment by grantfitzsimmons** [comment]
In Specify, there is not a way to send a notice to agents who have overdue loans. It is difficult to send these agents a reminder or keep on top of this managemnet.

There should be some way to send a reminder to these agents manually or automatically, maybe by creating a template that can be sent out quickly.

Requested By: Ana (Herpatology Collection Manager at KU)
